src/components/new-request/search.vue

```diff
@@ -678,6 +772,11 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
   padding: 20px 8px !important;
 }
 
+// Change color of the open form icon
+.open-form-icon {
+  color: $app-blue;
+}
```

Comment:
An alternate way to do this (and might be better here since there are no other class properties) is to set `color="primary"` in the v-icon.

Reply:
Looks good.